### PR TITLE
Update UI kit to v1.4 prerelease

### PR DESCRIPTION
**Pull request by maxious**
v1.4 will include some user facing changes to the in page navigation and typography. This branch is tracking changes in UI kit development so we can easily integrate when the final version is released. We are also removing custom SCSS where integrated upstream

update again after v1.4 final release.

uikitv1 4

### CODE REVIEW

File: lib/tasks/ui-kit.rake [hunk 1]
```diff
@@ -4,10 +4,10 @@ namespace :ui_kit do
 
   task :update do
     def download_asset(name, destination)
-      ui_kit_url = "http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/"
+      ui_kit_url = "http://gov-au-ui-kit-staging.apps.staging.digital.gov.au/latest/"
```
Comment (tessereth) [inline lib/tasks/ui-kit.rake]: Do we want this to pull in ui-kit develop or their last release?

**Comment by tessereth** [comment]
Apart from the question about the rake task, LGTM

**Comment by ltankey** [comment]
LGTM

**Comment by maxious** [comment]
rebase'd

**Pull request merged**